plugins/services/src/js/components/modals/ServiceDestroyModal.js

```diff
@@ -77,41 +84,26 @@ class ServiceDestroyModal extends React.Component {
   }
 
   handleRightButtonClick() {
-    this.props.deleteItem(this.shouldForceUpdate());
-  }
-
-  getDestroyMessage() {
     const { service } = this.props;
-    let serviceName = "";
+    const serviceName = service.getId();
+    const serviceNameConfirmationValue = this.state
+      .serviceNameConfirmationValue;
 
-    if (service) {
-      serviceName = service.getId();
+    if (serviceNameConfirmationValue === serviceName) {
+      this.props.deleteItem(this.shouldForceUpdate());
     }
+  }
 
-    if (service instanceof Framework) {
-      return (
-        <p>
-          This will only
-          {" "}
-          {UserActions.DELETE}
-          {" "}
-          the package scheduler for
-          {" "}
-          <span className="emphasize">{serviceName}</span>
-          . Any tasks that were started by this scheduler will persist in the system. Are you sure you want to continue?
-        </p>
-      );
-    }
+  handleChangeInputFieldDestroy(event) {
+    const { service } = this.props;
+    const serviceName = service.getId();
+    const inputValue = event.target.value;
+    const isButtonDisabled = inputValue === serviceName;
```

Comment:
Sorry, but this name still doesn't reflect the value. 
When service name equals the input value the button is enabled. We either should go for `inputValue !== serviceName` or change the name to `isButtonEnabled ` in either case we should than update line 105